style/standard.js

```diff
@@ -115,52 +108,6 @@ module.exports = {
-        "@typescript-eslint/no-unused-vars": ["error", { "ignoreRestSiblings": true, "varsIgnorePattern": "^__" }],     // Disallows unused vars, unless __Type;
-        "@typescript-eslint/prefer-nullish-coalescing": "error",                                            // Prefer "??" over "||", so we don't do a falsy check
-        "@typescript-eslint/prefer-readonly": "error",                                                      // Prefer readonly private members where possible
-        "@typescript-eslint/quotes": ["error", "double", { "allowTemplateLiterals": true }],                // Use "" quotes instead of '', or ``
-        "@typescript-eslint/type-annotation-spacing": "error",                                              // const foo: number, space after ":"
-        "@typescript-eslint/switch-exhaustiveness-check": "error",                                          // Requires all branches to be covered or a default
-        "@typescript-eslint/tslint/config": [                                                               // Legacy tslint rules, runs our semicolon rules
-            "error",
-            {
-                "rules": {
-                    "whitespace": [
-                        true,
-                        "check-branch",
-                        "check-decl",
-                        "check-operator",
-                        "check-separator",
-                        "check-type",
-                        "check-type-operator",
-                        "check-preblock"
-                    ],
-                    "ordered-imports": true,
-                    "semicolon": [true, "always"],
-                },
-            },
-        ],
-        "@typescript-eslint/typedef": [
-            "error",
-            {
-                "arrayDestructuring": false,                                                                // Infer types on array destructuring
-                "arrowParameter": true,
-                "memberVariableDeclaration": true,
-                "objectDestructuring": false,                                                               // Infer types on object destructuring
-                "parameter": true,
-                "propertyDeclaration": true,
-                "variableDeclaration": true,
-                "variableDeclarationIgnoreFunction": true,
-            }
-        ],
-        "@typescript-eslint/explicit-function-return-type": "error",                                        // force functions to declare return type
         "arrow-parens": ["error", "always"],                                                                // Force parens around arrow arguments
         "brace-style": ["error", "allman", { "allowSingleLine": true }],                                    // Parenthetically correct braces
         "comma-dangle": [                                                                                   // Force dangling commas if }/]/) appears on newline
```

Comment:
Issues in style that are actually correctness:
 - no-extend-native
 - no-async-promise-executor
 - no-console
 - no-duplicate-imports
 - no-useless-escape
 - no-useless-call
 - no-useless-catch
 - no-useless-computed-key
 - prefer-const
 - import/no-default-export
 - no-fallthrough
 - no-implicit-coercion
 - eqeqeq